.vscode/settings.json

```diff
@@ -0,0 +1,23 @@
+{
+  // Formatting
+  // Uncomment the following 2 lines below to allow Prettier to auto format your code on save 👇
+  // "notebook.formatOnSave.enabled": true,
+  // "editor.formatOnSave": true,
```

Comment:
I use it too. It's a part of my user settings. I commented it out and made a comment on how to enable it in the README because feel like this one is a bit of personal preference. Some people might not like their editor to format their code like this. Dunno, I'm happy to uncomment it if you think we should.

Reply:
Works for me